docs/tutorials/send-payjoin-with-pdk.md

```diff
@@ -0,0 +1,403 @@
+# Introduction
+
+This tutorial goal is to get you started with payjoin and demonstrate
+payjoin capabilities. We will demonstrate how to setup the sender side
+in a payjoin transaction and transact with a payjoin receiver
+utilising `payjoin-cli`.
+
+While `rust-payjoin` supports both BiP0078 and BiP0077, this tutorial
+is specificly to demonstrate BiP0078(payjoin version 1).
+
+For a quick wallet setup, we utilise `docker` and `bitcoincore_rpc`
+crate to run a regtest node and fund two wallets, one for the sender
+and one for the receiver.
+
+For any feedback, please free to reach out through the payjoin
+official discord or github.
```

Comment:
Any call to action should also be completed with hyperlinks. Don't make the reader do more work than strictly necessary. This should be a welcoming tutorial after all